geowebcache/core/src/test/resources/org/geowebcache/config/geowebcache_1100.xml

```diff
@@ -1,212 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
You seem to have simply deleted this file rather than moving it like you did for other versions.